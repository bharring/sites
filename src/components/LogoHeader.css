h1 {
  font-size: 36pt;
  font-family: 'Roboto';
}
