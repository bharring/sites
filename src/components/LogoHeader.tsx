import './LogoHeader.css';

import { IonCol, IonRow, IonText } from '@ionic/react';

const LogoHeader: React.FC = () => {
  return (
    <IonRow className="ion-padding-top ion-padding-bottom">
      <IonCol>
        <IonText className="ion-text-center" color="primary">
          <h1>Application Name</h1>
        </IonText>
      </IonCol>
    </IonRow>
  );
};
export default LogoHeader;
