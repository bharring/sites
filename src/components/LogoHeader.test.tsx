import '@testing-library/jest-dom';

import renderer from 'react-test-renderer';

import LogoHeader from './LogoHeader';

describe('<LogoHeader />', () => {
  test('renders LogoHeader snapshot', () => {
    const tree = renderer.create(<LogoHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
