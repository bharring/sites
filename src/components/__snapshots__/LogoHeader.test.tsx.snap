// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoHeader /> renders LogoHeader snapshot 1`] = `
<ion-row>
  <ion-col>
    <ion-text
      color="primary"
    >
      <h1>
        Application Name
      </h1>
    </ion-text>
  </ion-col>
</ion-row>
`;
